docs/documentation/siddhi-4.0.md

```diff
@@ -390,6 +390,12 @@ per "<TIME_PERIOD>"
 select a.<attributeName>, a.<attributeName>, a.<attributeName> 
 insert into fooBar;
 ```
+The above syntax includes the following:
+Item|Description
+---------|---------
+`within "<START_TIME>", "<END_TIME>"`|This allows you to specify the time interval for which the aggregate values need to be retrieved by specifying the timestamps for the start time and the end time.
```

Comment:
There's another way of defining within. We can give within as a single string (e.g. 2014-02-** **:**:** +05:30 or 2014-02-** **:**:**). For this too, the formats <yyyy>-<MM>-<dd>, <HH>:<mm>:<ss>, <Z> (if time is not in GMT), and <yyyy>-<MM>-<dd> <HH>:<mm>:<ss> (if time is in GMT) apply.
When 2014-02-** **:**:** is given, it conveys that we want to retrieve values within the 2nd month of 2014. That is, the time duration from 2014-02-01 00:00:00 to 2014-03-01 00:00:00 (excluding 2014-03-01 00:00:00)